model/nic.rb

```diff
@@ -27,4 +27,8 @@ def self.ubid_to_name(ubid)
   def ubid_to_tap_name
     ubid.to_s[0..9]
   end
+
+  def unlock
+    Semaphore.where(strand_id: strand.id, name: "lock").all.map(&:destroy)
```

Comment:
Does unlock ever has its value used? Isn't strand the strand for this nic? seems like two simplifications may be possible:

```ruby
 Semaphore.where(strand_id: id, name: "lock").delete
```

I had this as each(&:destroy) at first, but do we really want `DeletedRecord` entries for even semaphores? If not, straight `DELETE` might be fine, and much faster.